### PR TITLE
PF-2817: enable auth0 up to preprod

### PR DESCRIPTION
**Pull request by yuhuyoyo**

### CODE REVIEW

File: tools/render-config.sh [hunk 1]
```diff
@@ -106,3 +106,13 @@ clientSecret=$(docker run --rm -e VAULT_TOKEN="${VAULT_TOKEN}" "${DSDE_TOOLBOX_D
...
+
+echo "Fetching Verily auth0 prod client id and client secrets"
+clientId=$(docker run --rm -e VAULT_TOKEN="${VAULT_TOKEN}" "${DSDE_TOOLBOX_DOCKER_IMAGE}" \
+            vault read -format json "${CLIENT_CRED_VAULT_PATH}" | \
+            jq -r '.data."verily-auth0-prod-client-id"')
+clientSecret=$(docker run --rm -e VAULT_TOKEN="${VAULT_TOKEN}" "${DSDE_TOOLBOX_DOCKER_IMAGE}" \
+            vault read -format json "${CLIENT_CRED_VAULT_PATH}" | \
+            jq -r '.data."verily-auth0-prod-client-secret"')
+./tools/client-credentials.sh "src/main/resources/verily_auth0_secret.json" "${clientId}" "${clientSecret}" \
+                              "rendered/verily_auth0_secret.json"
```
Comment (dexamundsen) [inline tools/render-config.sh:118]: nit: new line 
also in 
src/main/resources/verily_auth0_secret.json

File: src/main/resources/servers/verily-autopush.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "verily-autopush",
-  "clientCredentialsFile": "verily_secret.json",
+  "clientCredentialsFile": "verily_auth0_dev_secret.json",
```
Comment (dexamundsen) [inline src/main/resources/servers/verily-autopush.json:3]: clarifying:
- dev secret for - dev, autopush and staging
- prod secret for - prod and preprod

File: src/main/resources/verily_auth0_secret.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (dexamundsen) [inline src/main/resources/verily_auth0_secret.json:1]: minor: rename file verily_auth0_secret to verily_auth0_prod_secret

**Review by cbookg — APPROVED** [review]

**Review by dexamundsen — APPROVED** [review]

File: src/main/resources/verily_auth0_prod_secret.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "installed": {
+    "auth_uri": "https://verily-terra-prod.us.auth0.com/authorize",
+    "token_uri": "https://verily-terra-prod.us.auth0.com/oauth/token",
+    "redirect_uris": [
+      "https://terra.verily.com/authcode"
+    ]
+  },
+  "description": "Secrets file. Client id & secrets are rendered to rendered/verily_auth0_prod_secret.json. This is used for preprod, prod."
+}
```
Comment (dexamundsen) [inline src/main/resources/verily_auth0_prod_secret.json:10]: nit - new line

File: .github/workflows/release-on-pr-merge.yml [hunk 5]
```diff
@@ -47,6 +47,7 @@ jobs:
         ./tools/client-credentials.sh "src/main/resources/broad_secret.json" ${{ secrets.BROAD_CLIENT_ID }} ${{ secrets.BROAD_CLIENT_SECRET }}
         ./tools/client-credentials.sh "src/main/resources/verily_secret.json" ${{ secrets.VERILY_CLIENT_ID }} ${{ secrets.VERILY_CLIENT_SECRET }}
         ./tools/client-credentials.sh "src/main/resources/verily_auth0_dev_secret.json" ${{ secrets.VERILY_AUTH0_DEV_CLIENT_ID }} ${{ secrets.VERILY_AUTH0_DEV_CLIENT_SECRET }}
+        ./tools/client-credentials.sh "src/main/resources/verily_auth0_secret.json" ${{ secrets.VERILY_AUTH0_PROD_CLIENT_ID }} ${{ secrets.VERILY_AUTH0_PROD_CLIENT_SECRET }}
```
Comment (dexamundsen) [inline .github/workflows/release-on-pr-merge.yml:50]: needs to be updated as well

**Pull request merged**